### PR TITLE
Add Web Share API

### LINKED ISSUE #496: Add data on web share api
Add the new web share api. Currently only available in chrome for android behind a origin trail.

https://developers.google.com/web/updates/2016/10/navigator-share

### PR DESCRIPTION
**Pull request by jbmoelker**
Resolves #496

This is my first contribution to this project. I didn't have too much documentation to go on. If there's anything you'd like me to change, I'm happy to do it. I'm a big fan of the Web Share API. I don't expect you to start development on it, I'd just like to have it on the radar and formally listed.

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -5157,5 +5157,20 @@
...
+    "name": "Web Share API",
+    "category": "Device",
+    "link": "https://wicg.github.io/web-share/",
+    "summary": "An API for sharing data (text, URLs, images) from the web to an app of the user's choosing.",
+    "standardStatus": "Editor's draft",
+    "spec": "web-share",
+    "ieStatus": {
+      "text": "Not currently planned"
+    },
+    "impl_status_chrome": "Enabled by default",
```
Comment (alrra) [inline status.json:5171]: Because `id` was provided, this can be removed.

(I'll do this change before merging).

**Comment by alrra** [comment]
@jbmoelker Thank you and congrats on your first contribution! 🎉

**Comment by molant** [comment]
@alrra i don't believe " not currently planned" is the right status. We are waiting for the spec to be more fleshed lut IIRC so this should be under consideration.

@kypflug can probably help with this.

**Comment by kypflug** [comment]
This is correct, it should be Under Consideration. I'll touch base with our web apps team on the priority but I know we're very interested in mapping this to the WinRT share contract on Windows.

**Comment by molant** [comment]
I've updated the status via e03b4ddbf790879a001bb65ff1b6e42ef84f1643 to reflect that.

**Pull request closed without merging**